git_machete/github.py

```diff
@@ -22,16 +23,44 @@ def __repr__(self) -> str:
         return f"PR #{self.number} by {self.user}: {self.head} -> {self.base}"
 
 
-GITHUB_TOKEN_ENV_VAR = 'GITHUB_TOKEN'
+GITHUB_TOKEN_ENV_VAR = "GITHUB_TOKEN"
+
+# GitHub Enterprise deployments use alternate domains.
```

Comment:
Whoops good point! I didn't think of that as I've never had a chance to use it yet... could you open an issue + reference it here via a `TODO`?